stories/SQFormDialogStepper.stories.js

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
-import {mixed, number, object, string} from 'yup';
+import * as Yup from 'yup';
+import Grid from '@material-ui/core/Grid';
 import {withKnobs, boolean} from '@storybook/addon-knobs';
 import {withInfo} from '@storybook/addon-info';
 import {action} from '@storybook/addon-actions';
+import {Typography} from '@material-ui/core';
```

Comment:
super nit: could we put all MUI imports together? And could we decide if we going to import from `/core` or not.
```
import Grid from '@material-ui/core/grid;
import {Typography} from `@material-ui/core`;
```